### PR TITLE
Add in publish profiles for both sites

### PR DESCRIPTION
**Pull request by s3364859**
@6nop @s3364942 @s3494110 @s3482247 

Nothing too exciting in here, added in file system publish profiles for both web sites.

collection

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -90,7 +90,7 @@ publish/
 *.azurePubxml
 
 # TODO: Comment the next line if you want to checkin your web deploy settings but database connection strings (with potential passwords) will be unencrypted
-*.pubxml
+# *.pubxml
```
Comment (s3364859) [inline .gitignore]: We actually want these files.

File: CPT331.Web/Properties/PublishProfiles/Release Web Publish.pubxml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--
+This file is used by the publish/package process of your Web project. You can customize the behavior of this process
+by editing this MSBuild file. In order to learn more about this please visit http://go.microsoft.com/fwlink/?LinkID=208121. 
+-->
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (s3364859) [inline CPT331.Web/Properties/PublishProfiles/Release Web Publish.pubxml]: Note that there are no passwords in here.

**Comment by s3364859** [comment]
I've added in those files and updated the gitignore file as well. Over to you @6nop!

**Comment by 6nop** [comment]
I've been looking forward to testing the automated build after this PR!

I'm happy with the changes :+1: 

Here we go!

_...And you'd better fire Team Services or an /uppercut is coming your way!_

**Pull request merged**